src/app/modules/form/form.service.ts

```diff
@@ -348,44 +350,74 @@ export const retrievePublicFormsWithSmsVerification = (
 
 export const createSingleSampleSubmissionAnswer = (field: FormFieldDto) => {
   let sampleValue = null
-  let noOfOptions = 0
-  let randomSelectedOption = 0
+  let noOfTableRows
+  let noOfTableCols
+  const tableSampleValue = []
   switch (field.fieldType) {
-    case 'textarea':
-    case 'textfield':
-      sampleValue =
-        'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.'
+    case BasicField.LongText:
+      sampleValue = faker.lorem.sentence()
       break
-    case 'radiobutton':
-    case 'dropdown':
-      noOfOptions = field.fieldOptions.length
-      randomSelectedOption = Math.floor(Math.random() * noOfOptions)
-      sampleValue = field.fieldOptions[randomSelectedOption]
+    case BasicField.ShortText:
+      sampleValue = faker.lorem.word()
```

Comment:
```suggestion
      sampleValue = faker.lorem.words()
```
I think `words` will be more reflective of what users will input in short answer fields too?

Reply:
[c91e6c5](https://github.com/opengovsg/FormSG/pull/6358/commits/c91e6c571c8c3b1ed13ab1eb543d84b111986842)